sdk/src/Core/Amazon.Util/Internal/_netstandard/InternalSDKUtils.netstandard.cs

```diff
@@ -57,8 +73,13 @@ public static string DetermineFramework()
         {
             try
             {
-                var desc = RuntimeInformation.FrameworkDescription.Trim();
-                return string.Format(CultureInfo.InvariantCulture, ".NET_Core#{0}", GetValidSubstringOrUnknown(desc, desc.LastIndexOf(' ') + 1, desc.Length));
+                var desc = _runtimeInformationWrapper.FrameworkDescription.Trim();
+                var version = GetValidSubstringOrUnknown(desc, desc.LastIndexOf(' ') + 1, desc.Length);
+                if (!Version.TryParse(version, out var _))
+                {
+                    return "Unknown" + $" md/framework-raw-version#{desc}";
```

Comment:
For the `desc` being added to there can be spaces in it which could mess up parsing logic assuming the value after the `#` has no spaces. Can you replaces spaces in `desc` with underscores? `desc.Replace(' ', '_');`